docs/DeveloperGuide.md

```diff
@@ -949,6 +1267,162 @@ Although all fields can be edited, the `Name`, `Amount`, and `Type` fields canno
 out in the **Add Transaction** section above also remain.
 </div>
 
+
+#### Clear Transactions
+
+**Overview**
+
+The `clear_transactions` command deletes all existing `Transactions` from `TransactionList` in UniCash.
+
+The activity diagram of clearing all transactions is as shown below
+
+<img src="images/unicash/diagrams/ClearTransactionsActivityDiagram.png" width="450" />
+
+The following sequence diagram shows the interaction between different components of UniCash.
```

Comment:
```suggestion
The following sequence diagram shows the interaction between different components of UniCa$h.
```

Reply:
There are a few more instances of "UniCash" being mentioned instead of "UniCa$h" under the get command section as well

Reply:
Fixed, thanks